llvm/test/Transforms/LoopVectorize/X86/uitofp-preserve-nneg.ll

```diff
@@ -0,0 +1,78 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt -passes="default<O3>,loop-vectorize" -force-vector-interleave=1 -force-vector-width=4 -S < %s | FileCheck %s
+
+target datalayout = "e-m:e-i64:64-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-unknown-linux-gnu"
```

Comment:
does this test need to be X86 specific? it looks like you are already forcing VF & interleave count, so would be good to drop the target triple and move to up one level if possible